index.php

```diff
@@ -277,11 +282,28 @@
         </form>
     </div>
     <!-- add record form -->
-
+    <!-- Filter records -->
+    <div class='filterContainer'>
+        <form method="POST" id='filterForm'>
```

Comment:
In this instance, you should actually switch to using the GET method - because there is no kind of security risk to having the genre in the URL, and it's related to getting data, GET is the way to go.

Another benefit of using GET is that you could actually link to the site with a filter applied